chia/full_node/mempool_manager.py

```diff
@@ -220,9 +174,8 @@ def __init__(
         if single_threaded:
             self.pool = InlineExecutor()
         else:
-            self.pool = ProcessPoolExecutor(
+            self.pool = ThreadPoolExecutor(
                 max_workers=2,
-                mp_context=multiprocessing_context,
```

Comment:
you  can remove the `multiprocessing_context` member now, as well as the initialization of it in `full_node.py`